app/project.godot

```diff
@@ -13,13 +13,35 @@ config_version=4
 config/name="jump-ship"
 run/main_scene="res://Scenes/MainMenu.tscn"
 config/icon="res://icon.png"
+<<<<<<<Updatedupstreamrun/main_scene="res://Scenes/MainMenu.tscn"
+<<<<<<<HEADrun/main_scene="res://Scenes/Main.tscn"
```

Comment:
shouldnt be adding this

Reply:
do I just go to project.godot and delete?

Reply:
yeah